app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -26,6 +28,15 @@ class EventViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
                     .into(itemView.eventImage)
         }
 
+        itemView.setOnClickListener{
+            val fragment = EventDetailsFragment()
+            val bundle = Bundle()
+            bundle.putLong("EVENT_ID", event.id)
```

Comment:
@iamareebjamal Should I create a Constant Values class and put EVENT_ID there and keep a reference here